openc3-init/plugins/packages/openc3-tool-limitsmonitor/src/tools/LimitsMonitor/LimitsControl.vue

```diff
@@ -307,13 +338,22 @@ export default {
     },
     update() {
       if (this.$store.state.tlmViewerItems.length !== 0) {
+        // localStorage.axiosIgnoreResponse = '500' // localStorage only supports strings
```

Comment:
This would potentially prevent popups on bad items but I think my config events handler mostly covers this now. Left it just in case as it's a useful pattern.

Reply:
This is a global setting, so probably dangerous?